### PR TITLE
discretisedfield

### PR DESCRIPTION
**Pull request by fangohr**
`discretisedfield` provides a (Python) class to represent (finite difference)  discretisation of a field (in the continuum field theory context).

It is part of the Jupyter-OOMMF project (https://github.com/joommf and http://joommf.github.io).

### CODE REVIEW

File: recipes/discretisedfield/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+build:
+  skip: True  # [py27]
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  build:
+    - python
+    - pip
+    - numpy >=1.11
```
Comment (synapticarbors) [inline recipes/discretisedfield/meta.yaml]: It doesn't look like anything besides `python` and `pip` are required to build the recipe. If that is the case please remove the extra build dependencies.

Comment (fangohr) [inline recipes/discretisedfield/meta.yaml]: Excuse my ignorance: what exactly happens in the build step? I had assumed that maybe pyc files are created, and thus imports will have to be carried out? Apologies if this is described somewhere - I couldn't find it.

Comment (isuruf) [inline recipes/discretisedfield/meta.yaml]: In the build step, the script given above (`pip install --no-deps .`) runs. Which means the files are installed and also `pyc` files are created. Creating pyc files doesn't care about imports.

Comment (fangohr) [inline recipes/discretisedfield/meta.yaml]: Thank you. I have removed everything apart from python and pip.

**Pull request merged**